src/App.tsx

```diff
@@ -1,25 +1,137 @@
-import React, { useState } from 'react';
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-console */
+import React, { useState, useEffect } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
 
 const App: React.FC = () => {
-  const [
-    selectedUserId,
-    // setSelectedUserId,
-  ] = useState(0);
+  const urlTodos = 'https://mate.academy/students-api/todos';
```

Comment:
move all login with fetch to separate file
create file api/api.ts and move fetch to this file